### PR TITLE
Add Security Center resources

### PR DESCRIPTION
**Pull request by irmatov**
* Auto Provisioning Settings
* Contacts
* Pricings
* Settings

### CODE REVIEW

**Review by fdistorted — COMMENTED** [review]
Looks good.

File: client/services/security.go [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+	// New*Client requires that the ASC location is passed as an argument, but API methods
+	// that we actually use do not use that location for performing the request (juding by REST API docs).
+	// That is why we are passing an empty string here.
+	aps := security.NewAutoProvisioningSettingsClient(subscriptionId, "")
+	aps.Authorizer = auth
+	pricings := security.NewPricingsClient(subscriptionId, "")
+	pricings.Authorizer = auth
+	contacts := security.NewContactsClient(subscriptionId, "")
+	contacts.Authorizer = auth
+	settings := security.NewSettingsClient(subscriptionId, "")
```
[the remark this answers is not in the document]
Comment (irmatov) [reply client/services/security.go:43]: Sure, thanks!

File: client/services/security.go [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+type SecurityClient struct {
+	AutoProvisioningSettings SecurityAutoProvisioningSettingsClient
+	Contacts                 SecurityContactsClient
+	Pricings                 SecurityPricingsClient
+	Settings                 SecuritySettingsClient
+}
+
+func NewSecurityClient(subscriptionId string, auth autorest.Authorizer) SecurityClient {
+	// New*Client requires that the ASC location is passed as an argument, but API methods
+	// that we actually use do not use that location for performing the request (juding by REST API docs).
```
Comment (roneli) [inline client/services/security.go:35]: juding -> judging

Comment (irmatov) [reply client/services/security.go]: Fixed, thanks

**Review by roneli — APPROVED** [review]

**Pull request merged**